version.php

```diff
@@ -19,15 +19,16 @@
  *
  * @package    format
  * @subpackage weeksrev
- * @copyright  2018 Arnaud Trouvé <moodle@arnaudtrouve.fr>
+ * @copyright  2024 1katoda
+ *            based on code by Arnaud Trouvé <moodle@arnaudtrouve.fr>
  *            based on code by Mat Cannings
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2019020300;        // The current plugin version (Date: YYYYMMDDXX).
-$plugin->requires  = 2017111300;        // Requires this Moodle version.
+$plugin->version   = 2025032600;        // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
Could you revert those changes please ? I'll take care to update it myself during release process.